frontend/src/component/common/Search/Search.tsx

```diff
@@ -102,20 +120,28 @@ export const Search = ({
                                         onSearchChange('');
                                         ref.current?.focus();
                                     }}
+                                    sx={{ p: 1 }}
```

Comment:
I'd do theme.spacing here. I've fixed a few bugs before where multiples of 1rem were mixed with 1px. With theme.spacing(1) we remove ambiguity